Rakefile

```diff
@@ -1,14 +1,16 @@
+# frozen_string_literal: true
```

Comment:
Is this a rubocop thing or a ruby parser thing?

Reply:
it's a ruby thing, see https://wyeworks.com/blog/2015/12/1/immutable-strings-in-ruby-2-dot-3

more specifically, it's a new rubocop suggested practice for apps using ruby > 2.3, and since this project seems to have a convention of following rubocop strictly with minimal config deviations, I decided to include it.